src/systems/systems/src/simulation/update_context.rs

```diff
@@ -21,10 +26,15 @@ pub trait DeltaContext {
 pub struct Attitude {
     pitch: Angle,
     bank: Angle,
+    true_heading: Angle,
```

Comment:
Just definition-wise I find it slightly questionable to consider true heading part of the attitude.

Reply:
Yeah agreed, I grouped the three here because you need those 3 to position the plane fully in space. I can just move heading at a single separate Angle indeed!